storagenode/storagenodedb/consoledb.go

```diff
@@ -33,10 +34,10 @@ func (db *consoledb) GetSatelliteIDs(ctx context.Context, from, to time.Time) (_
 
 	var satellites storj.NodeIDList
 
-	rows, err := db.db.QueryContext(ctx, db.Rebind(`
+	rows, err := db.QueryContext(ctx, `
```

Comment:
The `Rebind()` function this was calling was an empty function so I didn't see a need to keep it around. Is there a reason it should be kept around?

Reply:
I don't think so, I was curious too why it was there in the first place!

Reply:
What as `Rebind` intended to do?

Reply:
Not sure, the function was empty. I feel like it was an old pattern propagated or put in place just to satisfy an interface contract.

Reply:
SQLite3 requires `?` bind markers, but PostgreSQL requires `$N` bind markers (e.g. `$1`, `$2`, etc). DBX's `Rebind()` method is a convenience to translate from `?` to `$N` if the underlying driver is PostgreSQL, so you can write queries once with `?` and be done with it (or you could, in theory, if there were more than like 2 queries that worked on both systems).

For code that's storagenode-only, it's probably fine to leave out `Rebind()` everywhere, but on the other hand it's good to have a pattern and be accustomed to using it so that we don't get surprised when we actually need it. 🤷‍♂